internal/app/parse.go

```diff
@@ -112,3 +112,18 @@ func (a *App) compileGlob(pattern string) ([]string, error) {
 
 	return matches, nil
 }
+
+func (a *App) compileGlobExclusion(inputFiles []string, excludeFiles []string) []string {
```

Comment:
this function is not compiling globs, you should change its name to something like `excludeFilesFromInput` or `calculateInputExcludeDiff`, or `filterInputFilesWithExclusion`, etc.

Reply:
will do "filterExcludedFiles" @orellazri